bin/filter_gtf_for_genes_in_genome.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+from __future__ import print_function
+import logging
+from itertools import groupby
+import argparse
+
+# Create a logger
+logging.basicConfig(format='%(name)s - %(asctime)s %(levelname)s: %(message)s')
+logger = logging.getLogger(__file__)
+logger.setLevel(logging.INFO)
+
+def is_header(line):
+    return line[0] == '>'
+
+
+def extract_fasta_seq_names(fasta_name):
+    """
+    modified from Brent Pedersen
+    Correct Way To Parse A Fasta File In Python
+    given a fasta file. yield tuples of header, sequence
+    from https://www.biostars.org/p/710/
+    """
+    "first open the file outside "
```

Comment:
Feels like this should be a comment line..?
```suggestion
    # first open the file outside
```